src/main/java/de/dhbw/karlsruhe/students/mailflow/external/infrastructure/email/CreateEmailHelper.java

```diff
@@ -15,11 +16,16 @@ private CreateEmailHelper() {}
 
   public static Email createEmailWithMessage(Message message) throws EmailParsingException {
     try {
-      return Email.create(message.getContent().toString(),
-          EmailMetadataFactory.withMessage(message).build(), null);
-    } catch (IOException | MessagingException e) {
+
+      String content = message.getContentType();
+
+      EmailMetadataFactory factory = EmailMetadataFactory.withMessage(message);
+
+      EmailMetadata metadata = factory.build();
```

Comment:
```suggestion
      EmailMetadata factory = EmailMetadataFactory.withMessage(message).build();
```
Typically you write it like this when working with factories